docs/widgets.md

```diff
@@ -59,6 +59,32 @@ Example := Window {
 }
 ```
 
+## `ToggleButton`
+
+The ToggleButton looks like a button, but will stay `pressed` when clicked and can then be toggled
+back to un-`pressed` state by clicking again.
+
+### Properties
+
+- **`checked`** (*bool*): Shows whether the button is toggled or not
```

Comment:
I wonder if this property is needed and if instead it would be cleaner (or more "minimal" in terms of API surface) to say that for a toggle button the "pressed" property remains true.

I understand that we still need an internal property to accurately reflected the state, but API wise wouldn't it be nice if just "pressed" was sufficient? Especially since the documentation of `ToggleButton` says that it will "stay pressed" when clicked.

Reply:
You're right. And if we have `checkable` it makes sense to have a `checked` property. (I wish it was called "sticky" instead of "checkable" and then pressed would probably be okay, but let's not go down that route ;-)